concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-935	935	Stampfen	to stomp	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-936	936	Springen	to jump	VIII. Verba	D. Gehen, Kommen u.s.w.	1829	JUMP
+Gabelentz-1892-1074-937	937	Tanzen	to dance	VIII. Verba	D. Gehen, Kommen u.s.w.	1879	DANCE
+Gabelentz-1892-1074-938	938	Straucheln	to stumble	VIII. Verba	D. Gehen, Kommen u.s.w.	3617	STUMBLE
+Gabelentz-1892-1074-939	939	Fallen	to fall	VIII. Verba	D. Gehen, Kommen u.s.w.	1280	FALL
+Gabelentz-1892-1074-940	940	Ertrinken	to drown	VIII. Verba	D. Gehen, Kommen u.s.w.	1904	DROWN
+Gabelentz-1892-1074-941	941	Schwimmen	to swim	VIII. Verba	D. Gehen, Kommen u.s.w.	1439	SWIM
+Gabelentz-1892-1074-942	942	Fliessen	to flow	VIII. Verba	D. Gehen, Kommen u.s.w.	2003	FLOW
+Gabelentz-1892-1074-943	943	Tröpfeln	to dribble	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-944	944	Schwellen	to swell	VIII. Verba	D. Gehen, Kommen u.s.w.	1573	SWELL
+Gabelentz-1892-1074-945	945	Sein	to be	VIII. Verba	E. Dasein, verweilen	1579	BE
+Gabelentz-1892-1074-946	946	Dasein (wo sein)	to be there	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-947	947	Bleiben	to stay	VIII. Verba	E. Dasein, verweilen	987	REMAIN
+Gabelentz-1892-1074-948	948	Wohnen	to reside, to dwell	VIII. Verba	E. Dasein, verweilen	1099	RESIDE
+Gabelentz-1892-1074-949	949	Sitzen	to sit (be sitting)	VIII. Verba	E. Dasein, verweilen	1416	SIT
+Gabelentz-1892-1074-950	950	Stehen	to stand (be standing)	VIII. Verba	E. Dasein, verweilen	1442	STAND
+Gabelentz-1892-1074-951	951	stellen	to put something somewhere	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-952	952	Biegen	to bend	VIII. Verba	E. Dasein, verweilen	2224	BEND (SOMETHING)
+Gabelentz-1892-1074-953	953	Legen	to lay something somewhere	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-954	954	Hängen (intransitiv)	to hang (intransitive)	VIII. Verba	E. Dasein, verweilen	2884	HANG
+Gabelentz-1892-1074-955	955	hängen (transitiv)	to hang (transitive)	VIII. Verba	E. Dasein, verweilen	357	HANG UP
+Gabelentz-1892-1074-956	956	Kleben	to stick	VIII. Verba	E. Dasein, verweilen	254	STICKING
+Gabelentz-1892-1074-957	957	Geschehen	to happen	VIII. Verba	E. Dasein, verweilen	248	HAPPEN
+Gabelentz-1892-1074-958	958	Vergeben	to forgive	VIII. Verba	E. Dasein, verweilen	1528	FORGIVE
+Gabelentz-1892-1074-959	959	Fehlen, nicht dasein	to be missing	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-960	960	Abreissen, trennen	to tear apart	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1735	TEAR (SHRED)
+Gabelentz-1892-1074-961	961	Abwischen	to wipe	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1454	WIPE
+Gabelentz-1892-1074-962	962	Anfangen	to begin	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3447	BEGIN OR START
+Gabelentz-1892-1074-963	963	Anklopfen	to knock	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2396	KNOCK
+Gabelentz-1892-1074-964	964	Anrühren	to touch	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1892	TOUCH
```

Comment:
Double-mapping, but not wrong.

Reply:
anrühren is a shorter touching than berühren.

Reply:
I immediately thought about baking here. I'd unmap it because we already have _berühren_ mapped to TOUCH.

Reply:
Oh, good thinking. The baking context did not occur to me at all, but it makes sense.